frontend/packages/client/src/App.sass

```diff
@@ -303,15 +305,16 @@ hr
 	.container > .navbar .navbar-brand
 		margin-left: 0 !important
 
+
 @media (max-width: 769px)
 	.section
 		padding-right: 1rem
 		padding-left: 1rem
 	.navbar-burger
 		display: block
 		width: 1.25rem
-	.navbar-brand
-		flex-direction: row-reverse
+		span
+			height: 2px !important
```

Comment:
I think this PR needs to be rebased off main because this seems like an unrelated change that's already in main branch

Reply:
maybe let's try to decouple `rebase` and feature work. Create separate PR to merge new changes from 'v1.6' into `notification` and then our local branch only pull from `notification` so PR should only contain the new feature changes. @mannynotfound  @linxiao-li  what do you think?

Reply:
opened a PR here to merge v1.6 to notifications: https://github.com/DapperCollectives/CAST/pull/603. above change is one of the diff carried over.